package.json

```diff
@@ -16,6 +16,7 @@
     "build-swagger": "yarn redocly build-docs https://api.clickhouse.cloud/v1 --output build/cloud/manage/api/swagger.html",
     "prep-from-local": "bash ./scripts/copy-clickhouse-repo-docs.sh -l $1",
     "autogenerate-settings": "bash ./scripts/settings/autogenerate-settings.sh",
+    "autogenerate-system-tables": "bash ./scripts/system-tables/autogenerate-system-tables.sh",
```

Comment:
`autogenerate-settings` is badly named, but it does both settings and functions, and should do system tables too in the same manner.